### ISSUE TITLE
Extensions (likely adBlock) conflict

### ISSUE THREAD

**Issue by mikemklau** [MewsSystems/public#57]
I have a urgent issue on “Euston-square-hotel.com” but it is a complicated one.
 
To put simply:
 
• I have added the script which can be seen in the header as directed.
• I have had to add a second script calling the distributor.open() function separately just to get the widget to work to avoid booking losses
• I was informed this is very inefficient as it creates a new object everytime
• Without this extra script the widget only works in incognito windows or when extensions are disabled.
• I BELIEVE it is “adblocker” which stops the widget working.
• Either way I have other sites which this widget works even with adblocker
 
 
To sum UP:
 
The script can be seen in the header of the page by viewing page source (ctrl+u) on chrome. 
The script only works in incognito or on extension-less browsers.

At the moment, I have commented out the header script which calls distributor.min.js and I have copied the contents of that script into a local file and called it from there and it works. So the issue seems to be that the script src=....distributor.min.js is only called when I have no extensions or on incognito. I can't figure out what it is

**Comment by pbambusek** [comment]
Hello,

I've checked the website and as it works correctly now, I can't check the broken state. Can you create and host somewhere a copy with the previous, not-working setup for us to review?

I've no recollection of case where adBlocker would block the whole Distributor script. It usually (and quite correctly) blocks only GTM scripts.

**Comment by mikemklau** [comment]
Thanks so much for responding so fast

I have set up staging environment:

https://umi360.staging.wpengine.com/eustonsquare/

feel free to take a look


Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

**Comment by pbambusek** [comment]
Thank you for the staging environment. It's, as I suspected, a problem of the initialization code placement. It must be placed inside the `<body>` tag as mentioned in the documentation, not in the `<head>`. Putting it into the proper place will resolve your issue.

**Comment by mikemklau** [comment]
Thanks Petr!!

I’m just glad it is an easy fix!

Enjoy the rest of your day!

Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows 10

________________________________
